docs/smart-contracts/getting-started/setup.mdx

```diff
@@ -84,7 +84,7 @@ The [Soroban CLI](http://github.com/stellar/soroban-cli) can execute Soroban con
 Install the [latest released version](https://github.com/stellar/soroban-cli/releases) of Soroban CLI using `cargo install`.
 
 ```sh
-cargo install --locked soroban-cli --version 21.0.0-rc.1
```

Comment:
Aren't we moving toward `stellar-cli`?

Reply:
yes, there's a separate pr for that. #633

Reply:
I can merge #633 but there will be multiple PRs to move all the docs over.

Reply:
I think it's fine if we go over #633 slowly. I noticed there are a few conflicts to resolve.

Reply:
I think it's going to take more than a few pass. I'd refer to keep the commits small so there are less merge conflicts.